tests/lib/rules/no-length-assertions.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var rule = require('../../../lib/rules/no-length-assertions');
+var RuleTester = require('eslint').RuleTester;
+
+var ruleTester = new RuleTester();
+ruleTester.run('no-length-assertions', rule, {
+  valid: [{
+    code: 'expect([1]).to.have.lengthOf(1);'
+  }, {
+    code: 'expect([1].length).to.be.equal(1);'
+  }, {
+    code: 'expect([1].length).to.equal(1);'
+  }],
```

Comment:
could you make sure that stuff like `expect('foo').to.have.length.above(2);` also still works without raising errors?